flybywiresim/aircraft
# fix: APU Bleed Pressure when APU off

Fixes #8009 

## Summary of Changes
Added a check to see if APU is in normal operation while rendering bleed pressure

## Screenshots

Before 
![Screenshot_4](https://github.com/flybywiresim/a32nx/assets/39380117/e4773e45-aaba-43d9-be22-64dbc4598b27)

After 
![Screenshot_5](https://github.com/flybywiresim/a32nx/assets/39380117/fbd1d784-7678-4cf0-9761-57757f549730)


## References
https://youtu.be/2SHR2ouBaGc
https://youtu.be/VUjfYWNijA8



Discord username (if different from GitHub):
KoruptedToon#7018

## Testing instructions
1. Load the A32Nx in Cold and Dark State
2. Turn on batteries and connect external power
3. Turn on ANY ADIRS. 
3. Go to APU Page and check bleed pressure. Should reflect XX when APU is turned off. 
4. Should function normally and display numerical values in case APU is turned on.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
